classes/Domain/Model/Airport.php

```diff
@@ -15,7 +15,7 @@ class Airport extends Eloquent implements AirportInformationDatabase
      * @return Airport
```

Comment:
Should we update docblocks here and in other places as well?

I would like to refer to https://github.com/zendframework/zend-diactoros/pull/37#issuecomment-107195929. 

What do you think, @BackEndTea?

Reply:
Yeah i'll get around to updating that.

I guess this is also a time to wonder how much the docblocks are needed, if we can paint the whole picture with type declarations.